workspace/utilities/com/site-credits.xsl

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: site-credits -->
+	<xsl:template name="site-credits">
+		<xsl:param name="text">
+			<xsl:choose>
+				<xsl:when test="$url-language = 'fr'">
+					<xsl:text>Crédits</xsl:text>
+				</xsl:when>
+				<xsl:otherwise>
+					<xsl:text>Credits</xsl:text>
+				</xsl:otherwise>
+			</xsl:choose>
+		</xsl:param>
+		<xsl:param name="ext-attr" />
+
+
+		<!-- ATTRIBUTES -->
+		<xsl:variable name="attr">
+			<add class="flex-shrink-0 overflow-hidden" />
+			<add class="relative cursor-pointer" />
+			<add class="transition-site-credits" />
```

Comment:
It's supposed to be missing as its only used to create custom transition in util-transitions.less
I figured it would be useful to have them already in place..

Reply:
@PascalPiche Should I keep it or should I remove it?

Reply:
remove it plz :)